src/functionalTest/kotlin/com/github/jengelman/gradle/plugins/shadow/JavaPluginsTest.kt

```diff
@@ -104,6 +104,8 @@ class JavaPluginsTest : BasePluginTest() {
     }
 
     assertThat(shadowConfig.artifacts.files).contains(shadowTask.archiveFile.get().asFile)
+
+    error("This is intended to fail.")
```

Comment:
It works as expected!

```
Run if [ "failure" != "success" ]; then
  if [ "failure" != "success" ]; then
    echo "Some build jobs failed."
    exit 1
  fi
  echo "All build jobs completed successfully."
  shell: /usr/bin/bash -e {0}
Some build jobs failed.
Error: Process completed with exit code 1.
```

https://github.com/GradleUp/shadow/actions/runs/16930748117/job/47975785113